lib/modules/manager/flux/__fixtures__/helmChart.yaml

```diff
@@ -0,0 +1,14 @@
+apiVersion: source.toolkit.fluxcd.io/v1
```

Comment:
can you join both fixtures and inline them to test? or will it cause code coverage issues?

Reply:
could be also inlined in _most_ tests, but they are also used in one test for `extractAllPackageFiles`, where file paths are currently expected in the results (https://github.com/renovatebot/renovate/pull/31962/files#diff-e4fcec4565b58234272751cf6a22e089c6e13b651bb297b4db19834442b124e1R960-R961). 

quick search shows that this seems to be related to #19056, similar situation as in the referenced discussion/comments there?

Reply:
so, how do we proceed here?

Reply:
It's ok to leave some fixtures if it's impossible or painful to inline them. We can ignore the automated "no new fixtures" test and merge anyway